lxd/instance/drivers/driver_qemu.go

```diff
@@ -3051,6 +3051,33 @@ func (d *qemu) addNetDevConfig(cpuCount int, busName string, qemuDev map[string]
 			pciSlotName = nicItem.Value
 		} else if nicItem.Key == "pciIOMMUGroup" {
 			pciIOMMUGroup = nicItem.Value
+		} else if nicItem.Key == "mtu" {
+			mtuStr = nicItem.Value
+		}
+	}
+
+	if d.localConfig["agent.rename_interfaces"] != "" {
+		mtuInt, err := strconv.Atoi(mtuStr)
```

Comment:
Should use `strconv.ParseUint(value, 10, 32)` here instead, so that we don't allow a 64 bit int to be overflowed when its casted to uint32 below.